src/components/ListItem.jsx

```diff
@@ -1,5 +1,56 @@
 import './ListItem.css';
+import { updateItem } from '../api';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+const currentTime = new Date();
+
+export function ListItem({ name, items, listToken }) {
+	const { isChecked, id, totalPurchases, dateLastPurchased } = items;
+
+	const currentDate = new Date();
+
+	if (isChecked) {
+		const currentTimeToSec = currentDate.getTime() / 1000;
+		const lastPurchaseToMilSec = dateLastPurchased.seconds * 1000;
+		const timeElapsed = currentTimeToSec - lastPurchaseToMilSec;
+
+		const timeToUncheck = 86400;
```

Comment:
It would be nicer if the number `86400` was assigned to a constant, something like `ONE_DAY_IN_SECONDS` so it's more obvious what this number is :slightly_smiling_face: 
Here is an example explaining this concept more in depth: https://jeffreyeverhart.com/2020/09/28/javascript-tips-using-named-constants-for-magic-numbers/

Reply:
Makes sense. Thanks also for the resource with the explanation :)